src/index.ts

```diff
@@ -4446,7 +4446,6 @@ function tsPlugin(options?: {
             if (!isBinding && node.left.type === 'TSTypeCastExpression') {
               node.left = this.typeCastToParameter(node.left)
             }
-            return node
```

Comment:
Could u use return super.toAssignable instead of delete the return syntax?

Reply:
Thank you for checking this PR!
I changed that.